cmd/puppeth/wizard.go

```diff
@@ -160,27 +160,6 @@ func (w *wizard) readDefaultInt(def int) int {
 	}
 }
 
-// readFloat reads a single line from stdin, trimming if from spaces, enforcing it
-// to parse into a float.
-func (w *wizard) readFloat() float64 {
-	for {
-		fmt.Printf("> ")
-		text, err := w.in.ReadString('\n')
-		if err != nil {
-			log.Crit("Failed to read user input", "err", err)
-		}
-		if text = strings.TrimSpace(text); text == "" {
-			continue
-		}
-		val, err := strconv.ParseFloat(strings.TrimSpace(text), 64)
-		if err != nil {
-			log.Error("Invalid input, expected float", "err", err)
-			continue
-		}
-		return val
-	}
-}
```

Comment:
I guess there are three other alternatives:

* just show the megacheck warning: this is somewhat counter-productive, eventually the megacheck log should be empty, to spot the errors more easily
* comment out the code: easy to understand, no problem with uncommenting
* use `var _ = (*wizard)(nil).readFloat` - to mark it as needed

Commenting out seems the most reasonable alternative here. 

PS: There might be even better way of writing these read functions, with less repetition, but probably not good idea in this PR.